.github/workflows/source-web-tool-scripts.yaml

```diff
@@ -11,20 +11,17 @@ on:
 name: Source web-tool scripts
 
 jobs:
-  Test-r-scripts:
+  R-CMD-check:
```

Comment:
why is this changed to `R-CMD-CHECK` here in this action?

Reply:
Thanks for noticing. I was sloppy when I copy-pasted from the R CMD check workflow. Fixed now.